.github/workflows/ci.yaml

```diff
@@ -0,0 +1,160 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+env:
+  FLUTTER_VERSION: '3.10.x'
+
+jobs:
+  analyze:
+    runs-on: ubuntu-20.04
```

Comment:
Do we still need to run this on 20.04? Couldn't it be 22.04?

Reply:
all green :+1: